src/main/webapp/app/overview/course-score-calculation.service.ts

```diff
@@ -30,12 +30,18 @@ export class CourseScoreCalculationService {
         let presentationScore = 0;
         for (const exercise of courseExercises) {
             const isExerciseFinished = !exercise.dueDate || exercise.dueDate.isBefore(moment());
+            const isAssessmentOver = !exercise.assessmentDueDate || exercise.assessmentDueDate.isBefore(moment());
             const isExerciseIncluded = exercise.includedInOverallScore !== IncludedInOverallScore.NOT_INCLUDED;
+
             if (isExerciseFinished && isExerciseIncluded) {
                 const maxPointsReachableInExercise = exercise.maxPoints!;
                 if (exercise.includedInOverallScore === IncludedInOverallScore.INCLUDED_COMPLETELY) {
                     maxPointsInCourse += maxPointsReachableInExercise;
                 }
+                // points are reachable if the exercise is released and the assessment is over --> It was possible for the student to get points
+                if (exercise.includedInOverallScore === IncludedInOverallScore.INCLUDED_COMPLETELY && isAssessmentOver) {
+                    reachableMaxPointsInCourse += maxPointsReachableInExercise;
+                }
```

Comment:
Something small but this can be added to the if statement above it.

Reply:
Like that? -> 05f11a4